### PR TITLE
feat: Subscribe to documents

### PR DESCRIPTION
**Pull request by ukstv**

### CODE REVIEW

File: src/three-idx.ts [hunk 1]
```diff
@@ -59,10 +60,12 @@ export class ThreeIDX {
   public docs: Record<string, TileDocument>
   public ceramic: CeramicApi
   protected _v03ID?: string
+  protected _subscriptionSet: SubscriptionSet
```
Comment (oed) [inline src/three-idx.ts:63]: We need to hold on to the handles, or this is for future if we decide to close cleanly?

**Review by oed — APPROVED** [review]
Looks good. Just had a question.

File: src/three-idx.ts (hunk 1 above)
Comment (ukstv) [reply src/three-idx.ts:63]: Future, and clarity of maintenance. We do not have to operate on the handles right now, considering docs are not unloaded.

Comment (ukstv) [reply src/three-idx.ts:63]: Good to merge @oed ?

Comment (oed) [reply src/three-idx.ts:63]: yep ✅

**Pull request merged**